README.md

```diff
@@ -74,7 +75,7 @@ Target problems are very expressive and thus work well as targets for reduction,
 ### Target Problems
 
 - Halting problem for the call-by-value lambda-calculus (`HaltL` in [`L/L.v`](theories/L/L.v))
-- Provability or satisfiability in First-Order Logic (all problems in [`FOL/FOL.v`](theories/FOL/FOL.v))
+- Validity, provability or satisfiability in First-Order Logic (all problems in [`FOL/FOL.v`](theories/FOL/FOL.v))
```

Comment:
I'm not sure whether we use the Oxford comma consistently. I prefer "Validity, provability, or satisfiability"

Reply:
will fix